### DISCUSSION TITLE
preview multiple images

### DISCUSSION THREAD

**Discussion by pjotrthegreat** [QL-Win/QuickLook#1242] [Ideas]
hi, absolutely love this program!

idk how difficult it would be to implement but it would be cool if you could preview multiple images, the way it works in OSX's preview. ie you select the images on your desktop or wherever, hit space, and it shows a larger preview

![Screen Shot 17-12-2022 at 04-42](https://user-images.githubusercontent.com/33701702/208222408-3d019e67-4a6d-4a05-8654-c10ac716e19e.png)
![Screen Shot 17-12-2022 at 04-48](https://user-images.githubusercontent.com/33701702/208222409-f46fde32-89d6-4149-a23d-e6edca1d2504.png)

anyways thanks for making this, windows really lacks in quality of life usabiliy tools such as this one, and thanks for providing it free of charge!

**Comment by Mac3DX** [comment]
Yes, one of the main benefits in Mac OS quick view.
Also, I've noticed some bad colors visualization over native Windows Viewer. Need to be fixed.
+1
